InnerEye/ML/lightning_base.py

```diff
@@ -289,7 +289,7 @@ def on_train_end(self) -> None:
         This hook is called at the very end of training. Use that to write the very last set of training and
         validation metrics from the StoringLogger to disk.
         """
-        self.read_epoch_results_from_logger_and_store(epoch=self.current_epoch)
+        self.read_epoch_results_from_logger_and_store(epoch=self.current_epoch-1)
```

Comment:
why is this necessary?

Reply:
Epochs are now incremented at a different point, without this change many tests are broken